webtool/test_api.py

```
@@ -301,19 +301,21 @@ def test_correct_invalid_name_400(client):
 
 def test_correct_file_starts_scoped_job(client, monkeypatch, mit_anbieter):
     calls = {}
-    def fake_start(project, cmd, cwd, kind):
-        calls["cmd"] = cmd; calls["kind"] = kind; calls["project"] = project
+    def fake_start(project, cmd, cwd, kind, **kw):
+        calls["cmd"] = cmd; calls["kind"] = kind; calls["project"] = project; calls["base"] = kw.get("base")
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)